src/core_plugins/kibana/public/discover/components/fetch_error/fetch_error.js

```diff
@@ -43,9 +44,26 @@ const DiscoverFetchError = ({ fetchError }) => {
 
     body = (
       <p>
-         You can address this error by editing the &lsquo;{fetchError.script}&rsquo; field
-         in <a href={url}>Management &gt; Index Patterns</a>,
-         under the &ldquo;Scripted fields&rdquo; tab.
+        <FormattedMessage
+          id="kbn.discover.fetchError.howToAddressErrorDescription"
+          defaultMessage="You can address this error by editing the {fetchErrorScript} field
+            in {managementLink}, under the {scriptedFields} tab."
+          values={{
+            fetchErrorScript: '&lsquo;' + fetchError.script + '&rsquo;',
```

Comment:
@tibmt @Bargs I don't think the quotes need escaping here. This is how it looks now:

![](https://user-images.githubusercontent.com/209966/49442838-f8894400-f7ca-11e8-8b28-30fe54029fa3.png)